database/db.js

```diff
@@ -24,6 +17,10 @@ const projects = {
 
   deleteProject:( project_id ) => {
     return db.none( deleteProject, [project_id] )
+  },
+
+  changeProjectName:( project_name, project_id) => {
+    return db.one( changeProjectName, [project_name, project_id] )
```

Comment:
I would question why you are creating a separate const `changeProjectName`, instead of just inlining it in the function. It is wayyy easier to read if the SQL is in the same function